### DISCUSSION TITLE
ImageMagick build failed

### DISCUSSION THREAD

**Discussion by ptiruvaipati** [ImageMagick/ImageMagick#7284] [Help] [answered]
### ImageMagick version

7.1.1-31

### Operating system, version and so on

Windows 10

### Description

Hi Team,

We trying to build the following solution after running configure.exe using Visual Studio 2019.

**IM7.StaticDLL.sln (Static Multi-threaded DLL runtimes)**

Running this solution in **Debug** with **x64**. 

Build was failed due to some errors.

Here I attached the errors. 
[ImageMagick-Build errors.ods](https://github.com/ImageMagick/ImageMagick/files/15188165/ImageMagick-Build.errors.ods)

I am following the below information to build the solution.

https://github.com/ImageMagick/ImageMagick-Windows

Can you please help me to fix this issue?

Regards,
Pavan

**Comment by dlemstra** [comment]
Can you give it another try with the latest commit of that repository. I forgot to copy a file when VisualMagick was moved to that repository.

**Reply by ptiruvaipati** [reply]
@dlemstra ,

I have tried with latest commit. We still see some errors. 

[ImageMagick-Build errors.ods](https://github.com/ImageMagick/ImageMagick/files/15217892/ImageMagick-Build.errors.ods)


Regards,
Pavan

**Reply by dlemstra** [reply, marked as answer]
That looks like an issue with `Gdiplus`. Have you tried upgrading the Windows SDK?

**Reply by ptiruvaipati** [reply]
Build issue is resolved after upgrading Windows SDK. Thank you!
